runtimes/aleph-debian-12-python/create_disk_image.sh

```diff
@@ -31,7 +31,7 @@ apt-get install -y --no-install-recommends --no-install-suggests \
 
 echo "Pip installing aleph-sdk-python"
 mkdir -p /opt/aleph/libs
-pip3 install --target /opt/aleph/libs 'aleph-sdk-python==0.7.0' 'fastapi~=0.103.1'
+pip3 install --target /opt/aleph/libs 'aleph-sdk-python==0.9.0' 'aleph-message==0.4.4' 'fastapi~=0.109.2'
```

Comment:
Better to not be so strict with version, use `~=` will be better than `==`

Reply:
In the case of the runtimes, we want to know and be able to communicate the precise version of every file we ship.